src/index.html

```diff
@@ -101,5 +93,19 @@ <h1>Settings</h1>
   <footer>
     <script type="text/javascript" src="../node_modules/cucumber-forge-report-generator/src/templates/scripts.js"></script>
     <script type="text/javascript" src="render.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.7/lib/darkmode-js.min.js"></script>
```

Comment:
let's leverage [`darkmode-js`](https://www.npmjs.com/package/darkmode-js) instead of a cdn grab, please